### PR TITLE
Public map displays to non-logged in users

### LINKED ISSUE #1366: Public map rendering seems borked
Likely this is some weirdness with how we are setting up mapboxGL....

https://www.hylo.com/public/map/?sortBy=updated for logged in user

Image

The same url for non-logged in user, incogntio

Image

Same url for non-logged in user, other browser

Image

The 'map' or WebGL warnings here might provide some clue to sorting this out

Image

### PR DESCRIPTION
**Pull request by thomasgwatson**
closes #1366 

Found another bug on closing detail pages in the non-logged-in public map, while working on this ticket. Will file a ticket.

### CODE REVIEW

**Comment by tibetsprague** [comment]
What does this ticket do? we want the map to display to non logged in users if they go to /public/map

**Comment by tibetsprague** [comment]
oh I see, it fixes display of the map

File: src/routes/PublicLayoutRouter/PublicLayoutRouter.scss [hunk 1]
```diff
@@ -67,14 +67,20 @@
...
   align-items: center;
   justify-content: center;
 }
 
-.center-column > div {
+.non-map-view {
+  padding-bottom: 100px;
+}
+
+.non-map-view > {
```
Comment (tibetsprague) [inline src/routes/PublicLayoutRouter/PublicLayoutRouter.scss:79]: do we really need both a .map-view and a .non-map-view class? seems like you could just use one or the other and have a default that then gets overridden by the one class being set?

Comment (thomasgwatson) [reply src/routes/PublicLayoutRouter/PublicLayoutRouter.scss:79]: I tried doing that; its effectively the center-column styling that is the default. However, there are some other stylings that depend on the center-column class being present, so I couldn't just replace it without replicating all of those as well. This ended up being the least amount of code change

**Review by tibetsprague — APPROVED** [review]

**Pull request merged**